sc-kpm/sc-common/CMakeLists.txt

```diff
@@ -1,20 +1,8 @@
-set(SC_COMMON_SRC ${CMAKE_CURRENT_LIST_DIR})
+file(GLOB_RECURSE SOURCES "*.c" "*.h")
```

Comment:
Please don't use it. CMake's build system is designed to automatically track dependencies between source files and headers. When you use `file(GLOB_RECURSE ...)`, CMake loses the ability to track these dependencies accurately. This can lead to unnecessary rebuilds of files that haven't actually changed, or it can fail to rebuild files that have changed. There's CONFIGURE_DEPENDS, but it's not used in this script, only supported on CMake 3.12+ and isn't as portable
https://stackoverflow.com/questions/32411963/why-is-cmake-file-glob-evil
https://discourse.cmake.org/t/is-glob-still-considered-harmful-with-configure-depends/808